packages/vitest/src/utils/workers.ts

```diff
@@ -0,0 +1,8 @@
+import { availableParallelism, cpus } from 'node:os'
+
+export function getWorkersCountByPercentage(percent: string) {
+  const maxWorkersCount = availableParallelism?.() ?? cpus().length
```

Comment:
This needs to be on the default export otherwise Node.js will fail at runtime

Reply:
I see. I will change node:os mock code.

Reply:
I mock default export of node:os.